blackbox/docs/protocols/http.txt

```diff
@@ -153,7 +153,7 @@ be passed to the request::
     ... EOF
     {
       "cols" : [ "date", "position" ],
-      "col_types" : [ 11, 9 ],
+      "col_types" : [ 1400, 600 ],
```

Comment:
This is also a breaking change - and not a small one. I think the admin-ui is for example affected.

Would it be possible to keep the old ids? E.g. separate precedence/id into two separate concerns internally

Reply:
See also: https://crate.io/docs/crate/reference/protocols/http.html#column-types

Reply:
Ahh, I didn't know we expose the ids 😞 Of course we can keep the old ids and add new ones for type precedence.

Reply:
Reverted back to old IDs and introduce new precedence ids.